.env.development.example

```
@@ -0,0 +1,68 @@
+
+# ===================================
+# Authentication
+# ===================================
+JWT_SECRET=dev-jwt-secret-change-for-production
+AUTH_SESSION_DURATION=7d
+
+# ===================================
+# Email (SES)
+# ===================================
+SES_FROM_EMAIL=no-reply@dev.finance-manager.com
+SES_FROM_NAME=Finance Manager Dev
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)